### PR TITLE
RUG re-design 2014

### PR DESCRIPTION
**Pull request by otobrglez**
Design:
- PSD: https://dl.dropboxusercontent.com/u/88844/rug_design.psd
- Resources: https://www.dropbox.com/sh/ufame6poo8fy5qw/AACylvf70yaKYCcHRKAqM-qya?dl=0

Tasks:
- [x] Added SASS support to jekyll via [jekyll-sass](https://github.com/noct/jekyll-sass)
- [x] Extracted GA code to partial
- [x] Removed old css code
- [x] Started ignoring .sass-cache folder in .gitignore
- [x] Added HTML5 metadata
- [x] Make SASS work with GH pages (noted by @mrfoto)
- [x] Implemented header design
- [x] Implemented "big-what" section
- [x] Implemented footer design
- [x] Implemented single-page desing
- [x] Implemented front-page design
- [x] [Stickers](http://www.rug.si/stickers/) page has to be added
- [x] Past meetup recap button has to lead (somewhere)?
- [x] Update github-pages [@mrfoto]
- [x] Opacity to first page post has to be added behind posts with images
- [ ] Increase size of videos in single posts (for each post)
- [x] Add subscribe to newsletter form
- [x] Merge / discuss - Make responsive #8 
- [x] Social links in the header should be social icons
- [ ] custom 404? maybe? https://help.github.com/articles/custom-404-pages/
- [x] Remove 'atom.xml'
- [ ] In header second link should get user to list of videos
- [ ] We need a place for partners / supporters / advertisers
- [x] Add Slack integration

### CODE REVIEW

**Comment by otobrglez** [comment]
@mrfoto @zigomir This branch was updated. Can someone please finish the remaining tree tasks? :beer:  ;)

**Comment by matixmatix** [comment]
hello it's me
i can help you out with site responsiveness if you're okay with that

**Comment by mrfoto** [comment]
Of course we are :+1:



--

Miha Rekar

http://mr.si/

**Comment by otobrglez** [comment]
Hey dude!

I'll talk to you on monday.

- Oto

File: _includes/bigwhat.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="big-what">
```
Comment (mrfoto) [inline _includes/bigwhat.html]: what

**Comment by mfilej** [comment]
A bi blo možno dobit boljšo kvaliteto tega ozadja?
2015-04-01 at 08 33
Tale ni niti blizu 1x quality, kaj šele 2x. :smile:

**Comment by mrfoto** [comment]
@otobrglez ma kontakt od dizajnerja - mu boš pisu?

**Comment by mfilej** [comment]
@mrfoto Ja, bom vprašal, ker rekonstruirat to sigurno vzame več časa kot napisat en mail. :smile:

**Comment by otobrglez** [comment]
@mfilej Tukaj je [RUG folder na DB](https://www.dropbox.com/sh/ufame6poo8fy5qw/AACylvf70yaKYCcHRKAqM-qya?dl=0), ki ga je dal designer.

**Comment by matixmatix** [comment]
@SloveniaRUG/owners a lahko prosim zmergamo branch "responsive", da ne bo potem kaj divergiralo...

**Comment by mfilej** [comment]
Sweet, hvala!

p.s. prozorna ozadja so :pig: :pig: :pig: Mi je koncno uspelo, ampak nisem znal drugače, kot dat vsem postom fiksno višino:
2015-04-01 at 10 29

**Comment by otobrglez** [comment]
@mfilej Pušaj commite, da pogledam če je kak drug way.

**Comment by otobrglez** [comment]
@mfilej Prosim če tudi mrđaš @matixmatix ov branch.

**Comment by mfilej** [comment]
@otobrglez Bom priparvil pull request. Kateri je @matixmatixov branch?

Aja, še to: kako je pa s fonti? Imamo kje tiste, ki jih je dizajner določil, ali gremo kar s klasiko (mogoče celo boljš, da ne nalaga se še tisto)?

**Comment by otobrglez** [comment]
@mfilej glej #8 font naj ostane ta ki je sedaj.

**Comment by mfilej** [comment]
Ok merged. To bo rebase party, haha. :tada:

**Comment by mfilej** [comment]
Ok ni blo hudega, ampak mogoce responsive branch ni bil ready?

2015-04-01 at 21 21

@matixmatix Mogoce ves kaj bi blo narobe al revertam?

**Comment by matixmatix** [comment]
ayyy pozabu pushat asset fajle :crying_cat_face:

**Comment by mfilej** [comment]
@matixmatix Ok ni problema, dej še en PR. :)

**Comment by matixmatix** [comment]
@mfilej done

**Comment by mfilej** [comment]
@matixmatix Dela, hvala. :+1:

**Comment by mrfoto** [comment]
Zakaj removat atom.xml? Če že, potem je treba RSS namest tega http://joelglovier.com/writing/rss-for-jekyll/

**Comment by mrfoto** [comment]
We use [Slackin](https://github.com/rauchg/slackin) on http://srug-invite.herokuapp.com/

**Comment by otobrglez** [comment]
@mrfoto Si naredil [slack.rug.si](http://slack.rug.si/)?

**Comment by mrfoto** [comment]
Ja

**Comment by mrfoto** [comment]
Rebased and ready to merge unless anyone has any final fixes? cc @SloveniaRUG/owners 

TODOs moved to #15. If you have more in mind just add there.

**Comment by mfilej** [comment]
Meni zgleda vse ok. :100:

**Pull request merged**